models/post-comment-model.js

```diff
@@ -0,0 +1,43 @@
+//REQUIRES
+const db = require("../db/connection.js");
+
+//FUNCTION
+function newComment(article_id, postingComment) {
+    let dateObj = new Date();
+    if (!postingComment.body) {
+        return Promise.reject({
+            status: 400,
+            msg: `Nothing to post!`
+        })
+    }
+    if (!postingComment.username) {
+        return Promise.reject({
+            status: 400,
+            msg: `No anonymous posting allowed, please log in`,
+      });
+    }
+    if (Object.keys(postingComment).length !==2) {
+        return Promise.reject({
+            status: 400,
+            msg: `Invalid comment posting request`,
+        });
+    } 
+        return db
+            .query(
+               `INSERT INTO comments (votes, created_at, author, body, article_id) 
```

Comment:
`votes` and `created_at` can be removed from the query...
- Votes will default to 0 
- created_at will be inserted automatically by psql